multiaddr/codecs/utf8.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import absolute_import
+
+from . import LENGTH_PREFIXED_VAR_SIZE
+
+
+SIZE = LENGTH_PREFIXED_VAR_SIZE
+IS_PATH = False
+
+
+def to_bytes(proto, string):
+    if len(string) == 0:
+        raise ValueError("empty ip6zone")
```

Comment:
Use `raise ValueError("{0} value must not be empty".format(proto.name))` here to make this generic over protocol names.

Reply:
Good catch! Thanks for the review:)